generators/server/templates/_pom.xml

```diff
@@ -504,6 +512,11 @@
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-test</artifactId>
```

Comment:
Hi, I have one question, Why did you add this dependency? It is already inside of spring-boot-starter-test, is there any reason to do that?